build_tools/sdk.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
```

Comment:
Trying to make it easier to keep the version numbers in a single file.
Also, the goal is to allow for finding tools on the local host as a fallback.
(I.e. not requiring to package the tools)